agents/nvt-fat-developer/.env.example

```
@@ -0,0 +1,41 @@
+# Samme oppsett som jr-/sr-agentene (se apps/llm-gateway/README.md).
+# AUTH_TOKEN er fat-devs EGEN konsument-nøkkel i gatewayens routes.json, slik
+# at trafikken kan skilles i loggen og nøkkelen revokeres alene. Det ekte
+# OAuth-tokenet bor kun i gatewayens .env og er aldri inne i denne
+# containeren. Modell-allowlisten håndheves i gatewayen (fail closed) —
+# agenten kan ikke velge en dyrere modell selv.
+#
+# ⚠️ Host-oppslaget må VERIFISERES i M0 (issue #96): nvt-runtimen kjører med
+# `network_mode: service:docker`, så det er ikke gitt at
+# host.docker.internal løses her. Er den ikke det, brukes gateway-IP-en.
+ANTHROPIC_BASE_URL=http://host.docker.internal:8787
+ANTHROPIC_AUTH_TOKEN=fat-developer
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANTHROPIC_AUTH_TOKEN key should go before the ANTHROPIC_BASE_URL key

(UnorderedKey)